components/pickerbutton/stories/pickerbutton.stories.js

```diff
@@ -108,35 +114,177 @@ export default {
 		isQuiet: false,
 		isDisabled: false,
 		isFocused: false,
-		isKeyboardFocused: false,
 		iconType: "ui",
 		iconName: "ChevronDown",
-		position: "right"
+		position: "right",
 	},
 };
 
+// Story specific templates
+const SizingTemplate = (args) => html`
+	<div
+		style=${styleMap({
+			display: "flex",
+			gap: "24px",
+			flexWrap: "wrap",
+		})}
+	>
+		${["s", "m", "l", "xl"].map(
+			(size) => html`
+				<div
+					style=${styleMap({
+						display: "flex",
+						gap: "16px",
+						flexDirection: "column",
+						alignItems: "center",
+						flexBasis: "80px",
+					})}
+				>
+					${Typography({
+						semantics: "detail",
+						size: "s",
+						content: [
+							{
+								s: "Small",
+								m: "Medium",
+								l: "Large",
+								xl: "Extra-large",
+							}[size],
+						],
+						customStyles: {
+							"white-space": "nowrap",
+							"--mod-detail-font-color": "var(--spectrum-seafoam-900)",
+						},
+					})}
+					${Template({ ...args, size })}
+				</div>
+			`,
+		)}
+	</div>
+`;
+
+const CustomIconTemplate = (args) => html`
+	<div
+		style=${styleMap({
+			display: "flex",
+			gap: "24px",
+			flexWrap: "wrap",
+		})}
+	>
+		<div
+			style=${styleMap({
+				display: "flex",
+				gap: "16px",
+				flexDirection: "column",
+				alignItems: "center",
+				flexBasis: "80px",
+			})}
+		>
+			${Typography({
+				semantics: "detail",
+				size: "s",
+				content: ["UI icon"],
+				customStyles: {
+					"white-space": "nowrap",
+					"--mod-detail-font-color": "var(--spectrum-seafoam-900)",
+				},
+			})}
+			${Template({
+				...args,
+				iconName: "ArrowDown100",
```

Comment:
Should we use the "ChevronDown" icon here? Or is that just a workflow icon?

Reply:
This part of the template is to show a custom UI icon (other than the default Chevron).